target/linux/realtek/dts-5.10/rtl8380_d-link_dgs-1210-10mp.dts

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "rtl838x_d-link_dgs-1210_soc.dtsi"
+#include "rtl83xx_d-link_dgs-1210_poe.dtsi"
+
+/ {
+	compatible = "d-link,dgs-1210-10mp", "realtek,rtl838x-soc";
+	model = "D-Link DGS-1210-10MP";
+};
+
+&ethernet0 {
+	mdio: mdio-bus {
+		compatible = "realtek,rtl838x-mdio";
+		regmap = <&ethernet0>;
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		INTERNAL_PHY(8)
```

Comment:
ugh, I guess this target already uses these macro's, but I strongly dislike them personally. Just write it out, we write code once, but read it many times, being specific here, means it's not up for interpretation later.

Reply:
@oliv3r I am not sure what you want me to do here?
```
#address-cells = <1>;
#size-cells = <0>;
```
Am I going to replace this with something and in that case what? Or am I supposed to remove it?

All devices in the family has exactly this code?

Reply:
I have looked in every dts and dtsi file there is for all realtek devices and every device have exactly this code so there is no problem.